components/Cases/__snapshots__/Cases.spec.jsx.snap

```diff
@@ -1,5 +1,50 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Cases component should render a error message when a person is restricted 1`] = `
```

Comment:
Always keep an eye on what the snapshots rendered, in this case it's a spinner 😅